### ISSUE TITLE
'getTransaction' overrides a member function but is not marked 'override'

### ISSUE THREAD

**Issue by pmitchev** [amjuarez/bytecoin#57]
Cannot compile v1.0.9 on Mac:
```
...bytecoin/tests/UnitTests/TestWalletService.cpp:905:29: error: 'getTransaction' overrides a member function but is not marked 'override'
      [-Werror,-Winconsistent-missing-override]
  virtual WalletTransaction getTransaction(size_t transactionIndex) const {
```

**Comment by amjuarez** [comment]
Thank you, as always, pmitchev.
We are going to fix it as soon as possible and will update the issue. 

In the meantime, you can simply add 'override' to 'virtual WalletTransaction getTransaction(size_t transactionIndex) const override {' line (we figured you've probably already done that, but just to make sure)

**Comment by amjuarez** [comment]
Fixed in 1.0.9.1

**Comment by forkadmins** [comment]
Hi May i know how to fix this error? This is coming frequently! Pls help

09:30:18.846551 INFO One of outputs for one of inputs have wrong tx.unlockTime = 2557
09:30:18.846972 INFO Failed to handle_output for output no = 1, with absolute offset 411
09:30:18.847202 INFO Failed to get output keys for tx with amount = 0.10000000 and count indexes 2
09:30:18.847431 INFO Failed to check ring signature for tx
09:30:18.847669 INFO Block has at least one transaction with wrong inputs:
09:30:18.850382 INFO One of outputs for one of inputs have wrong tx.unlockTime = 2557
09:30:18.850619 INFO Failed to handle_output for output no = 1, with absolute offset 411
09:30:18.850840 INFO Failed to get output keys for tx with amount = 0.10000000 and count indexes 2
09:30:18.851053 INFO Failed to check ring signature for tx
09:30:18.851289 INFO Failed to switch to alternative blockchain
09:30:18.877986 INFO Rollback success.
09:30:18.878320 INFO The block was inserted as invalid while connecting new alternative chain, block_id:
09:30:18.881213 INFO One of outputs for one of inputs have wrong tx.unlockTime = 2557
09:30:18.881444 INFO Failed to handle_output for output no = 1, with absolute offset 411
09:30:18.881683 INFO Failed to get output keys for tx with amount = 0.10000000 and count indexes 2
09:30:18.881895 INFO Failed to check ring signature for tx
09:30:32.526921 INFO [207.200.39.124:47371 INC] Sync data returned unknown top block: 2553 -> 3325 [772 blocks (1 days) behind]
09:30:32.526921 INFO SYNCHRONIZATION started
09:30:33.802385 INFO ----- BLOCK ADDED AS ALTERNATIVE ON HEIGHT 2553
09:30:33.802385 INFO id:
09:30:33.802385 INFO PoW:
09:30:33.802385 INFO difficulty: 2958175
09:30:33.830582 INFO ###### REORGANIZE on height: 2553 of 2553 with cum_difficulty 706025926
09:30:33.830582 INFO alternative blockchain size: 2 with cum_difficulty 708998576
09:30:33.861243 INFO One of outputs for one of inputs have wrong tx.unlockTime = 2557
09:30:33.861640 INFO Failed to handle_output for output no = 1, with absolute offset 411
09:30:33.861890 INFO Failed to get output keys for tx with amount = 0.10000000 and count indexes 2
09:30:33.862121 INFO Failed to check ring signature for tx
09:30:33.862342 INFO Block has at least one transaction with wrong inputs:
09:30:33.865292 INFO One of outputs for one of inputs have wrong tx.unlockTime = 2557
09:30:33.865543 INFO Failed to handle_output for output no = 1, with absolute offset 411
09:30:33.865788 INFO Failed to get output keys for tx with amount = 0.10000000 and count indexes 2
09:30:33.866013 INFO Failed to check ring signature for tx
09:30:33.866263 INFO Failed to switch to alternative blockchain
09:30:33.896247 INFO Rollback success.
09:30:33.896579 INFO The block was inserted as invalid while connecting new alternative chain, block_id:
09:30:33.899543 INFO One of outputs for one of inputs have wrong tx.unlockTime = 2557
09:30:33.899832 INFO Failed to handle_output for output no = 1, with absolute offset 411
09:30:33.900077 INFO Failed to get output keys for tx with amount = 0.10000000 and count indexes 2
09:30:33.900291 INFO Failed to check ring signature for tx
09:30:35.710713 INFO [207.200.39.124:14824 OUT] Sync data returned unknown top block: 2553 -> 3325 [772 blocks (1 days) behind]
09:30:35.710713 INFO SYNCHRONIZATION started
09:30:36.626295 INFO ----- BLOCK ADDED AS ALTERNATIVE ON HEIGHT 2553
09:30:36.626295 INFO id:
09:30:36.626295 INFO PoW:
09:30:36.626295 INFO difficulty: 2958175
09:30:36.654739 INFO ###### REORGANIZE on height: 2553 of 2553 with cum_difficulty 706025926
09:30:36.654739 INFO alternative blockchain size: 2 with cum_difficulty 708998576
09:30:36.683631 INFO One of outputs for one of inputs have wrong tx.unlockTime = 2557
09:30:36.684010 INFO Failed to handle_output for output no = 1, with absolute offset 411
09:30:36.684223 INFO Failed to get output keys for tx with amount = 0.10000000 and count indexes 2
09:30:36.684434 INFO Failed to check ring signature for tx
09:30:36.684652 INFO Block has at least one transaction with wrong inputs:
09:30:36.687320 INFO One of outputs for one of inputs have wrong tx.unlockTime = 2557
09:30:36.687550 INFO Failed to handle_output for output no = 1, with absolute offset 411
09:30:36.687768 INFO Failed to get output keys for tx with amount = 0.10000000 and count indexes 2
09:30:36.687978 INFO Failed to check ring signature for tx
09:30:36.688211 INFO Failed to switch to alternative blockchain
09:30:36.715267 INFO Rollback success.
09:30:36.715678 INFO The block was inserted as invalid while connecting new alternative chain, block_id:
09:30:36.718844 INFO One of outputs for one of inputs have wrong tx.unlockTime = 2557
09:30:36.719084 INFO Failed to handle_output for output no = 1, with absolute offset 411
09:30:36.719294 INFO Failed to get output keys for tx with amount = 0.10000000 and count indexes 2
09:30:36.719502 INFO Failed to check ring signature for tx
09:30:38.020910 INFO [207.200.39.124:38462 INC] Sync data returned unknown top block: 2553 -> 3326 [773 blocks (1 days) behind]
09:30:38.020910 INFO SYNCHRONIZATION started
09:30:39.291963 INFO ----- BLOCK ADDED AS ALTERNATIVE ON HEIGHT 2553
09:30:39.291963 INFO id:
09:30:39.291963 INFO PoW:
09:30:39.291963 INFO difficulty: 2958175
09:30:39.319453 INFO ###### REORGANIZE on height: 2553 of 2553 with cum_difficulty 706025926
09:30:39.319453 INFO alternative blockchain size: 2 with cum_difficulty 708998576
09:30:39.347976 INFO One of outputs for one of inputs have wrong tx.unlockTime = 2557
09:30:39.348295 INFO Failed to handle_output for output no = 1, with absolute offset 411
09:30:39.348522 INFO Failed to get output keys for tx with amount = 0.10000000 and count indexes 2
09:30:39.348775 INFO Failed to check ring signature for tx
09:30:39.349036 INFO Block has at least one transaction with wrong inputs:
09:30:39.351669 INFO One of outputs for one of inputs have wrong tx.unlockTime = 2557
09:30:39.351898 INFO Failed to handle_output for output no = 1, with absolute offset 411
09:30:39.352121 INFO Failed to get output keys for tx with amount = 0.10000000 and count indexes 2
09:30:39.352332 INFO Failed to check ring signature for tx
09:30:39.352592 INFO Failed to switch to alternative blockchain
09:30:39.378542 INFO Rollback success.
09:30:39.378839 INFO The block was inserted as invalid while connecting new alternative chain, block_id:
09:30:39.381771 INFO One of outputs for one of inputs have wrong tx.unlockTime = 2557
09:30:39.382000 INFO Failed to handle_output for output no = 1, with absolute offset 411
09:30:39.382208 INFO Failed to get output keys for tx with amount = 0.10000000 and count indexes 2
09:30:39.382417 INFO Failed to check ring signature for tx
09:30:39.798897 INFO [207.200.39.124:41621 INC] Sync data returned unknown top block: 2553 -> 3326 [773 blocks (1 days) behind]
09:30:39.798897 INFO SYNCHRONIZATION started
09:30:41.006239 INFO ----- BLOCK ADDED AS ALTERNATIVE ON HEIGHT 2553
09:30:41.006239 INFO PoW:
09:30:41.006239 INFO difficulty: 2958175
09:30:41.033594 INFO ###### REORGANIZE on height: 2553 of 2553 with cum_difficulty 706025926
09:30:41.033594 INFO alternative blockchain size: 2 with cum_difficulty 708998576
09:30:41.062218 INFO One of outputs for one of inputs have wrong tx.unlockTime = 2557
09:30:41.062566 INFO Failed to handle_output for output no = 1, with absolute offset 411
09:30:41.062792 INFO Failed to get output keys for tx with amount = 0.10000000 and count indexes 2
09:30:41.063005 INFO Failed to check ring signature for tx
09:30:41.063226 INFO Block has at least one transaction with wrong inputs:
09:30:41.065894 INFO One of outputs for one of inputs have wrong tx.unlockTime = 2557
09:30:41.066137 INFO Failed to handle_output for output no = 1, with absolute offset 411
09:30:41.066369 INFO Failed to get output keys for tx with amount = 0.10000000 and count indexes 2
09:30:41.066586 INFO Failed to check ring signature for tx
09:30:41.066832 INFO Failed to switch to alternative blockchain
09:30:41.093053 INFO Rollback success.
09:30:41.093383 INFO The block was inserted as invalid while connecting new alternative chain, block_id:
09:30:41.096312 INFO One of outputs for one of inputs have wrong tx.unlockTime = 2557
09:30:41.096552 INFO Failed to handle_output for output no = 1, with absolute offset 411
09:30:41.096797 INFO Failed to get output keys for tx with amount = 0.10000000 and count indexes 2
09:30:41.097011 INFO Failed to check ring signature for tx
09:30:45.287827 INFO [207.200.39.124:49047 INC] Sync data returned unknown top block: 2553 -> 3326 [773 blocks (1 days) behind]
09:30:45.287827 INFO SYNCHRONIZATION started
09:30:46.547723 INFO ----- BLOCK ADDED AS ALTERNATIVE ON HEIGHT 2553
09:30:46.547723 INFO id:
09:30:46.547723 INFO PoW:
09:30:46.547723 INFO difficulty: 2958175
09:30:46.578160 INFO ###### REORGANIZE on height: 2553 of 2553 with cum_difficulty 706025926
09:30:46.578160 INFO alternative blockchain size: 2 with cum_difficulty 708998576
09:30:46.609496 INFO One of outputs for one of inputs have wrong tx.unlockTime = 2557
09:30:46.609977 INFO Failed to handle_output for output no = 1, with absolute offset 411
09:30:46.610282 INFO Failed to get output keys for tx with amount = 0.10000000 and count indexes 2
09:30:46.610594 INFO Failed to check ring signature for tx
09:30:46.610892 INFO Block has at least one transaction with wrong inputs:
09:30:46.613928 INFO One of outputs for one of inputs have wrong tx.unlockTime = 2557
09:30:46.614193 INFO Failed to handle_output for output no = 1, with absolute offset 411
09:30:46.614416 INFO Failed to get output keys for tx with amount = 0.10000000 and count indexes 2
09:30:46.614654 INFO Failed to check ring signature for tx
09:30:46.614920 INFO Failed to switch to alternative blockchain
09:30:46.643611 INFO Rollback success.
09:30:46.644058 INFO The block was inserted as invalid while connecting new alternative chain, block_id:
09:30:46.647004 INFO One of outputs for one of inputs have wrong tx.unlockTime = 2557
09:30:46.647260 INFO Failed to handle_output for output no = 1, with absolute offset 411
09:30:46.647481 INFO Failed to get output keys for tx with amount = 0.10000000 and count indexes 2
09:30:46.647704 INFO Failed to check ring signature for tx
09:30:52.992019 INFO [207.200.39.124:35857 INC] Sync data returned unknown top block: 2553 -> 3326 [773 blocks (1 days) behind]
09:30:52.992019 INFO SYNCHRONIZATION started
09:30:54.165655 INFO ----- BLOCK ADDED AS ALTERNATIVE ON HEIGHT 2553
09:30:54.165655 INFO id:
09:30:54.165655 INFO PoW:
09:30:54.165655 INFO difficulty: 2958175
09:30:54.193106 INFO ###### REORGANIZE on height: 2553 of 2553 with cum_difficulty 706025926
09:30:54.193106 INFO alternative blockchain size: 2 with cum_difficulty 708998576
09:30:54.221599 INFO One of outputs for one of inputs have wrong tx.unlockTime = 2557
09:30:54.221967 INFO Failed to handle_output for output no = 1, with absolute offset 411
09:30:54.222193 INFO Failed to get output keys for tx with amount = 0.10000000 and count indexes 2
09:30:54.222462 INFO Failed to check ring signature for tx
09:30:54.222681 INFO Block has at least one transaction with wrong inputs:
09:30:54.225308 INFO One of outputs for one of inputs have wrong tx.unlockTime = 2557
09:30:54.225525 INFO Failed to handle_output for output no = 1, with absolute offset 411
09:30:54.225795 INFO Failed to get output keys for tx with amount = 0.10000000 and count indexes 2
09:30:54.226047 INFO Failed to check ring signature for tx
09:30:54.226287 INFO Failed to switch to alternative blockchain
09:30:54.252537 INFO Rollback success.
09:30:54.252881 INFO The block was inserted as invalid while connecting new alternative chain, block_id:
09:30:54.255755 INFO One of outputs for one of inputs have wrong tx.unlockTime = 2557
09:30:54.255993 INFO Failed to handle_output for output no = 1, with absolute offset 411
09:30:54.256220 INFO Failed to get output keys for tx with amount = 0.10000000 and count indexes 2
09:30:54.256444 INFO Failed to check ring signature for tx
09:30:55.894760 INFO [207.200.39.124:14824 OUT] Sync data returned unknown top block: 2553 -> 3326 [773 blocks (1 days) behind]
09:30:55.894760 INFO SYNCHRONIZATION started
09:30:56.753057 INFO ----- BLOCK ADDED AS ALTERNATIVE ON HEIGHT 2553
09:30:56.753057 INFO id:
09:30:56.753057 INFO PoW:
09:30:56.753057 INFO difficulty: 2958175
09:30:56.780808 INFO ###### REORGANIZE on height: 2553 of 2553 with cum_difficulty 706025926
09:30:56.780808 INFO alternative blockchain size: 2 with cum_difficulty 708998576
09:30:56.809650 INFO One of outputs for one of inputs have wrong tx.unlockTime = 2557
09:30:56.810259 INFO Failed to handle_output for output no = 1, with absolute offset 411
09:30:56.810497 INFO Failed to get output keys for tx with amount = 0.10000000 and count indexes 2
09:30:56.810737 INFO Failed to check ring signature for tx
09:30:56.810995 INFO Block has at least one transaction with wrong inputs:
09:30:56.813759 INFO One of outputs for one of inputs have wrong tx.unlockTime = 2557
09:30:56.814004 INFO Failed to handle_output for output no = 1, with absolute offset 411
09:30:56.814241 INFO Failed to get output keys for tx with amount = 0.10000000 and count indexes 2
09:30:56.814471 INFO Failed to check ring signature for tx
09:30:56.814735 INFO Failed to switch to alternative blockchain
09:30:56.841029 INFO Rollback success.
09:30:56.841468 INFO The block was inserted as invalid while connecting new alternative chain, block_id:
09:30:56.844567 INFO One of outputs for one of inputs have wrong tx.unlockTime = 2557
09:30:56.844841 INFO Failed to handle_output for output no = 1, with absolute offset 411
09:30:56.845070 INFO Failed to get output keys for tx with amount = 0.10000000 and count indexes 2
09:30:56.845297 INFO Failed to check ring signature for tx <d853e1018faa46d085f5f1f65bc8c69dbc89b4d031ad980bd2cd52f28c8f540e
